unit_test/sparse/Test_Sparse_block_gauss_seidel.hpp

```diff
@@ -67,34 +67,67 @@ typedef Kokkos::complex<double> kokkos_complex_double;
 typedef Kokkos::complex<float> kokkos_complex_float;
 
 using namespace KokkosKernels;
+using namespace KokkosKernels::Impl;
 using namespace KokkosKernels::Experimental;
 using namespace KokkosSparse;
 using namespace KokkosSparse::Experimental;
 
 namespace Test {
 
-template <typename crsMat_t, typename vector_t, typename const_vector_t,
-          typename device>
+enum GSApplyType {
+  symmetric,
+  forward_sweep,
+  backward_sweep,
+};
+
+template <typename lno_t, typename scalar_t, typename mag_t>
+struct GSTestParams {
+  // Intentionally testing block_size that's not a multiple of #rows.
+  lno_t block_size = 7;
+  lno_t numVecs    = 2;  // how many columns X/Y have
+  scalar_t omega   = 0.9;
+  mag_t tolerance  = 1e-7;  // relative error for solution x vector
+
+  // variants to cover
+  std::vector<GSAlgorithm> gs_algorithms = {
+      GS_DEFAULT,  // == GS_TEAM
+#if 0
```

Comment:
Clean-up please

Reply:
Transformed disabled code to notes in comments to make it clear which algorithms are covered.